src/command_modules/azure-cli-iot/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.2.1
++++++
+* Removed deprecated commands which have moved to the iot extension
```

Comment:
This should be version 0.3.0 since removing commands (even deprecated ones) is a breaking change. It should also be annotated as such.

Reply:
ditto

Reply:
ok sounds good